subsys/bluetooth/controller/ll_sw/nordic/lll/lll_central_iso.c

```diff
@@ -408,10 +409,30 @@ static int prepare_cb(struct lll_prepare_param *p)
 				/* sn and nesn are 1-bit, only Least Significant bit is needed */
 				cis_lll->sn += cis_lll->tx.bn * cis_lazy;
 				cis_lll->nesn += cis_lll->rx.bn * cis_lazy;
+
+				/* Adjust sn and nesn for canceled events */
+				if (err) {
+					/* Adjust sn when flushing Tx */
+					/* FIXME: When Flush Timeout is implemented */
+					if (cis_lll->tx.bn_curr <= cis_lll->tx.bn) {
+						lll_flush_tx(cis_lll);
+					}
+
+					/* Adjust nesn when flushing Rx */
+					/* FIXME: When Flush Timeout is implemented */
```

Comment:
These two `FIXME` does not contain enough/any information for anyone else to know how to fix.

Reply:
I guess that's OK. Generally it would be nice if they could refer to an existing PR or GH Issue so that someone looking at the code don't end up scratching their heads too much :D